docs/sources/datasources/tempo/_index.md

```diff
@@ -26,6 +26,10 @@ For instructions on how to add a data source to Grafana, refer to the [administr
 Only users with the organization administrator role can add data sources.
 Administrators can also [configure the data source via YAML](#provision-the-data-source) with Grafana's provisioning system.
 
+This video explains how to add data sources, including Loki, Tempo, and Mimir, to Grafana and Grafana Cloud. Tempo data source set up starts at 4:58 in the video.
+
+{{< youtube id="cqHO0oYW6Ic" start="298" >}}
+
```

Comment:
@jdbaldry I tried this but it didn't seem to work on the local build. I'm not going to worry about it. We have the note in the text on where to start. We can figure it out later and iterate on it.

Reply:
Bah, sorry. This will work on the website. I need to release a new version of the `grafana/docs-base` image.